system/default_ad_api/src/interface_version.cpp

```diff
@@ -0,0 +1,38 @@
+// Copyright 2022 TIER IV, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "default_ad_api/nodes/interface_version.hpp"
+
+namespace default_ad_api
+{
+
+InterfaceVersionNode::InterfaceVersionNode(const rclcpp::NodeOptions & options)
+: Node("interface_version", options)
```

Comment:
Is it possible to add some unit tests? I believe it's much better than manual testing.

Reply:
Add test. https://github.com/autowarefoundation/autoware.universe/pull/704/commits/67a63b4260b4ffcd88fbc07b6d2789eb5ca68538